cpu/stm32f7/include/periph_cpu.h

```diff
@@ -28,6 +28,15 @@ extern "C" {
 #endif
 
 /**
+ * @name    Starting address of the CPU ID
+ */
+#ifdef CPU_MODEL_STM32F722ZE
+#define CPUID_ADDR          (0x1ff07A10)
```

Comment:
Minor nit: everything is lowercase except the `A`

Reply:
copy pasta, will fix